server/utils/FirebaseAlgoliaServer/functions/tests/integrationTestHelpers.js

```diff
@@ -0,0 +1,311 @@
+// Initialize constants, libraries
+// These are public values 
+const config = {
+                  apiKey: "AIzaSyCR9KXK2iDuaZ3n6WNzZnBORbtpgWE7dvI",
+                  authDomain: "step9-2020-capstone.firebaseapp.com",
+                  projectId: "step9-2020-capstone"
+                };
+
+const admin = require('firebase-admin');
+admin.initializeApp(config);
+
+const db = admin.firestore();
+const COLLECTION = 'ads'; 
+
+const algoliasearch = require('algoliasearch');
+const functions = require('firebase-functions');
+
+const APP_ID = functions.config().algolia.app;
+const ADMIN_KEY = functions.config().algolia.key;
+
+// initialize algoliasearch API
+const client = algoliasearch(APP_ID, ADMIN_KEY);
+const index = client.initIndex('dev_ADS'); // current index
+
+const DEFAULT_SLEEP = 5000; // Firebase->Algolia takes a couple secs, WCS
+
+// Firebase helper functions 
+// N.B.: A lot of verbose names ahead. Not really sure if better names
+// Exist for most of these. 
+
+/**
+ * Add a json entity to COLLECTION 
+ * @param {json=} jsonEntity the entity to be added to firestore
+ * @return {!Promise}  
+ */
+async function addEntity(jsonEntity) {
+  const entityPromise = await db.collection(COLLECTION).add(jsonEntity);
+
+  await sleep(DEFAULT_SLEEP); // Give the addition some time to reach Algolia
+  console.log(`Entity added with id ${entityPromise.id}`);
```

Comment:
Don't need these logs.  Either the test works or it doesn't.  If it doesn't, you can put logs in locally to debug.